src/library/F3DOptions.cxx

```diff
@@ -58,27 +59,10 @@ class ConfigurationOptions
   }
 
   template<class T>
-  static std::string ToString(T currValue)
+  static std::string ToString(const T& currValue)
```

Comment:
I'm not against it, but this class will be reworked from the ground up, or even removed in this form

Reply:
That's fine if we remove it later. It's just a clean-up.